lib/ReactViews/Custom/Chart/Legends.jsx

```diff
@@ -52,14 +50,40 @@ class Legend extends React.PureComponent {
     const squareSize = 20;
     const Glyph = Glyphs[this.props.glyph] ?? GlyphCircle;
     return (
-      <LegendItem margin="0 5px">
-        <svg width={`${squareSize}px`} height={`${squareSize}px`}>
+      <Label title={label.text}>
+        <svg
+          width={`${squareSize}px`}
+          height={`${squareSize}px`}
+          style={{ flexShrink: 0 }}
+        >
           <Glyph top={10} left={10} fill={label.value} size={100} />
         </svg>
-        <LegendLabel className={Styles.label} align="left" margin="0 0 0 4px">
-          {label.text}
-        </LegendLabel>
-      </LegendItem>
+        <LabelText>{label.text}</LabelText>
+      </Label>
     );
   }
 }
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: auto;
+  padding: 7px;
+  font-size: 0.8em;
+`;
+
+const Label = styled.div`
```

Comment:
could use a <Text /> component here. An optional thing might do will be to add an aria-label for this element which is               id={sanitizeIdString(chartItem.key + 'label')}, then we can add in an aria-labelledby on the chart itself.

Reply:
Can you clarify this? The `Label` here is not form label although it is called `Label`. Maybe I should rename it to something else less confusing. I like the suggestion to use `aria-label`.